packages/stockflux-launcher/public/searchResultsWindow.css

```diff
@@ -65,7 +65,12 @@ body {
   background-color: transparent;
   cursor: pointer;
   outline: none;
-  color: #95989C;
+  color: #37d8bc;
```

Comment:
Is there a hover state for the buttons? Would you be able to add a screenshot if so?

Reply:
Maybe we should create a variable for the color if it does not exist yet?